planning/motion_velocity_planner/autoware_motion_velocity_planner/CMakeLists.txt

```
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.14)
+project(autoware_motion_velocity_planner_node)
+
+find_package(autoware_cmake REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+
+rosidl_generate_interfaces(
+  ${PROJECT_NAME}
+  "srv/LoadPlugin.srv"
+  "srv/UnloadPlugin.srv"
+  DEPENDENCIES
+)
+
+autoware_package()
+
+ament_auto_add_library(${PROJECT_NAME}_lib SHARED
+  DIRECTORY src
+)
+
+rclcpp_components_register_node(${PROJECT_NAME}_lib
+  PLUGIN "autoware::motion_velocity_planner::MotionVelocityPlannerNode"
+  EXECUTABLE ${PROJECT_NAME}_exe
```

```suggestion
  EXECUTABLE ${PROJECT_NAME}_node
```
is preferred I think.

---

@maxime-clem Wouldn’t it be better to change the PR title to `autoware_motion_velocity_planner` ~_node~? I think it’s sufficient if the node name has a `_node` suffix. https://github.com/autowarefoundation/autoware_core/pull/242#discussion_r2013387062

---

For the executable name using `_node` instead of `_exe` is fine.

Since this PR only adds one package of the motion velocity planner, I think it is more clear to keep the `_node` suffix. Otherwise it becomes unclear which packages this PR is adding.

---

Since there is a possibility that nodes and utilities will be added to this package, adding the `_node` suffix to the package name actually feels more confusing.

---

This is the only node package so I do not see a problem with the `_node`  suffix.

The Motion Velocity Planner feature is made of one node (package added by this PR), one library package (`_common` package), and multiple module packages. See https://github.com/autowarefoundation/autoware_universe/tree/main/planning/motion_velocity_planner.

Sorry if this is not very clear.